br/pkg/restore/client.go

```diff
@@ -823,6 +846,132 @@ func (rc *Client) createTablesInWorkerPool(ctx context.Context, dom *domain.Doma
 	return eg.Wait()
 }
 
+// CheckTargetClusterFresh check whether the target cluster is fresh or not
+// if there's no user dbs or tables, we take it as a fresh cluster, although
+// user may have created some users or made other changes.
+func (rc *Client) CheckTargetClusterFresh(ctx context.Context) error {
```

Comment:
@WangLe1321 maybe we can reuse this function in PiTR check requirements.

Reply:
[use GetExistedUserDBs](https://github.com/pingcap/tidb/pull/35397/commits/7f63c75d249ccd1581be83092b26fa51481a2de4)